swift/Workflow/Sources/WorkflowNode.swift

```diff
@@ -2,7 +2,7 @@ import ReactiveSwift
 import Result
 
 /// Manages a running workflow.
-final class WorkflowNode<WorkflowType: Workflow> {
+final internal class WorkflowNode<WorkflowType: Workflow> {
```

Comment:
`internal` is implicit, generally prefer leaving it out (or, for consistency, add it everywhere... but that doesn't sound great)

Reply:
Took it back off. It was mostly that I was walking through all these types, it was a helpful signal to me of what is and isn't visible.

I agree with this overall (leaving it off, since it is implicit. Adding it everywhere does seem like it too much noise)